tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -1,80 +1,136 @@
 from __future__ import division
 
+import mock
+import numpy as np
+import random
+import tempfile
 import unittest
 
+from chainer import serializers
 from chainer import testing
 from chainer import training
 
 
-class DummyUpdater(training.Updater):
+def get_trainer_with_mock_updater(iter_per_epoch):
```

Comment:
`chainer.testing.get_trainer_with_mock_updater` uses `IntervalTrigger` internally. I think using `chainer.testing.get_trainer_with_mock_updater` for testing `IntervalTrigger` is not good.